Makefile

```diff
@@ -81,6 +82,19 @@ DEVICE_INFO := -DDEVICE_NR_BUS_BITS=3 \
 
 DEVICE_LIBS += -lmemio
 DEVICE_INCLUDES += -I/usr/local/include/memio
+else ifeq ($(USE_CHIP2CHIP_DEVICE), 1)
+#Chip2chip Device's Setting
+DEVICE_INFO := -DDEVICE_NR_BUS_BITS=3 \
+	       -DDEVICE_NR_CHIPS_BITS=3 \
+	       -DDEVICE_NR_PAGES_BITS=7 \
+	       -DDEVICE_NR_BLOCKS_BITS=19 \
+	       -DUSER_MODE \
+	       -DUSE_PMU \
+	       -DUSE_KTIMER \
+	       -DUSE_NEW_RMW \
+	       -D_LARGEFILE64_SOURCE \
+	       -D_GNU_SOURCE \
+	       -DNOHOST
```

Comment:
This is not needed. AFAIK, these flags use for the BlueDBM machine. Please refer to the Ramdisk setting.